common/main/sgs.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
+<!-- Copyright © 1991-2024 Unicode, Inc.
+For terms of use, see http://www.unicode.org/copyright.html
+SPDX-License-Identifier: Unicode-DFS-2016
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+
+Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/ for derived annotations.
+-->
+<ldml>
+	<identity>
+		<version number="$Revision$"/>
+		<language type="sgs"/>
+        <script type="Latin"/>
+		<territory type="LT"/>
```

Comment:
```suggestion
```

drop these from this file

Reply:
@srl295 but why? Should it be in other way said or this information is needless?

Reply:
I missed this question - but it's incorrect in this file. This file is `sgs`.xml so it has no script or territory.